views/components/tabular-data.html

```diff
@@ -0,0 +1,29 @@
+{% macro tabularData(myLatest) %}
```

Comment:
`myLatest` is not a great name for the content being used. We want to use as few words as possible, as clear as possible, with as little camelCase as possible. As the macro is going to be called `table`, its argument should be something like `data`